src/test/java/org/mockitousage/junitrule/StubbingWarningsJUnitRuleTest.java

```diff
@@ -0,0 +1,257 @@
+package org.mockitousage.junitrule;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.Statement;
+import org.mockito.Mock;
+import org.mockito.internal.junit.JUnitRule;
+import org.mockito.internal.util.SimpleMockitoLogger;
+import org.mockitousage.IMethods;
+import org.mockitoutil.TestBase;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class StubbingWarningsJUnitRuleTest extends TestBase {
+
+    private SimpleMockitoLogger logger = new SimpleMockitoLogger();
+    private JUnitRule jUnitRule = new JUnitRule(logger, false);
+    private FrameworkMethod dummy = mock(FrameworkMethod.class);
```

Comment:
`dummy` should be renamed to `mock` otherwise it is not clear in your tests that dummy is a mock